armi/cases/tests/test_suiteBuilder.py

```diff
@@ -16,6 +16,51 @@
 Unit tests for the SuiteBuilder
 """
 import unittest
+from armi.cases.inputModifiers.inputModifiers import SamplingInputModifier
+from armi import cases, settings
+from armi.cases.suiteBuilder import LatinHyperCubeSuiteBuilder
+
+
+cs = settings.Settings("armi/cases/tests/anl-afci-177.yaml")
+case = cases.Case(cs)
```

Comment:
loading the settings like this probably why you had to copy the blueprints and other files into the test directory. it would be nice if we could avoid that altogether (at least from the repository's point of view). could you try copying the inputs as part of the test setup, from within a TemporaryDirectoryChanger (https://github.com/terrapower/armi/blob/d1fc198dc422faa53151791b13485c24f497ad81/armi/utils/directoryChangers.py#L195)?

See https://github.com/terrapower/armi/blob/07a06e6e35d85d6be6bab4d95887007ad6a09173/armi/bookkeeping/tests/test_databaseInterface.py#L88 and https://github.com/terrapower/armi/blob/eeb28b46de305cf2477ca92fd2aec811704fdd13/armi/cases/tests/test_cases.py#L107 as examples